metadata_annotation.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<blockquote> are queried</blockquote><p> </p><p>can be queried</p><p> </p><p>maybe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>